packages/playground/src/domains/agent-builder/components/agent-edit/agent-profile/__tests__/agent-step-container.test.tsx

```
@@ -0,0 +1,75 @@
+// @vitest-environment jsdom
+import { cleanup, fireEvent } from '@testing-library/react';
+import { afterEach, beforeEach, describe, expect, it } from 'vitest';
+
+import { StreamRunningContext } from '../../../../contexts/stream-chat-context';
+import { useWizard, type WizardStep } from '../../../../contexts/wizard-context';
```

ESLint

[error] Prefer using a top-level type-only import instead of inline type specifiers.

(import/consistent-type-specifier-style)